qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -143,7 +147,11 @@ def update_calibrations(self, experiment_data: ExperimentData):
 
 
 class FineXAmplitudeCal(FineAmplitudeCal):
-    """A calibration experiment to calibrate the amplitude of the X schedule."""
+    """A calibration experiment to calibrate the amplitude of the X schedule.
```

Comment:
Same here and all the other subclasses of FineAmplitude and other subclassed experiments

Reply:
We can add analysis_ref here but this means if we update characterization analysis then one should note analysis ref is also here, which is not automatically detected. This is kind of soft reference through experiment class itself.

Reply:
I'm fine with that. This is for benefit of people reading the API docs.